study.md

```diff
@@ -134,15 +134,15 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, give a brief description of what HTTP is.
 
 ```md
-<!-- your answer here -->
+Long ago, I memorized HTTP to stand for "Hyper Text Transfer Protocol". I understand it to be a standard from which different browser applications can use to interpret and display data.
```

Comment:
Yes it is one set of rules that govern how client's and servers send requests and responses to each other.